nhsx/nhsx-website
# Change NHS logo and Site brand text

This commit makes some minor changes to the branding of the NHSx site changing the logo to the NHS England logo, ensuring that it displays at 60px high, and changing the Site name text from "NHS England - Transformation Directorate" to "Transformation Directorate"

To test:
- view any page on the local site and confirm that the logo now displays NHS England, rather than just NHS
- Confirm that the Site title displayed is now Transformation Directorate.

You may need to clear the cache in the admin section to see the changes
